src/VisualStudio/InteractiveServices/Interactive/VsResetInteractive.cs

```diff
@@ -259,5 +263,15 @@ protected override IWaitIndicator GetWaitIndicator()
         {
             return _componentModel.GetService<IWaitIndicator>();
         }
+
+        /// <summary>
+        /// Return namespaces that can be resolved in the latest interactive compilation.
+        /// </summary>
+        protected override async Task<IEnumerable<string>> GetNamespacesToImport(IEnumerable<string> namespacesToImport, IInteractiveWindow interactiveWindow)
+        {
+            var document = interactiveWindow.CurrentLanguageBuffer.CurrentSnapshot.GetOpenDocumentInCurrentContextWithChanges();
```

Comment:
I did not find an easier way to get the compilation using just `IInteractiveWindow` and `IInteractiveEvaluator` interfaces.